index.js

```diff
@@ -523,15 +528,6 @@ var compile = function(schema, cache, root, reporter, opts) {
   validate = validate.toFunction(scope)
   validate.errors = null
 
-  validate.__defineGetter__('error', function() {
```

Comment:
The standards compliant way would be to use `defineProperty` which isn't supported in IE<11 unfortunately. 

I opted for simply adding the error property to validate after trying a simple function that acted as a getter first. There were scope issues with that first attempt. I couldn't get validate.errors to be anything but null in this context unless I aloud for validate.errors to be called as a function.